src/components/LinkedAccounts/PlaidLinkError/plaidLinkError.scss

```diff
@@ -0,0 +1,19 @@
+.Layer__plaid-link-error__content {
+  container-type: inline-size;
+
+  .Layer__plaid-link-error__data-state {
+    .Layer__data-state__description {
+      margin-bottom: 0;
+    }
+  }
+
+  @container (max-width: 26.25rem) {
```

Comment:
Where does this number come from? Asking to see if it's a repeatable constant we should re-use as a var.